lib/Doctrine/CouchDB/HTTP/SocketClient.php

```diff
@@ -290,6 +290,7 @@ public function request($method, $path, $data = null, $raw = false, array $heade
                     while ($bytesLeft > 0) {
                         $body .= $read = fread($this->connection, $bytesLeft + 2);
                         $bytesLeft -= strlen($read);
+                        $body = substr($body, 0, -2);
```

Comment:
After review I'm understand that trimming is not good idea. In 2 line above we read 2 additional symbols and we need to trim they only.
Otherwise it may trim spaces/tabs/etc that actualy was in that json.